publish_release.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+DLL=JotunnModStub/bin/Release/JotunnModStub.dll
+PLUGINS=ChebsNecromancy/Package/plugins
```

Comment:
Please replace this with ChebsNecromancy with the generic name

Reply:
Fixed, thanks